api/proto/buf.yaml

```diff
@@ -27,6 +27,8 @@ lint:
     - teleport/legacy/types/types.proto
     - teleport/legacy/types/wrappers/wrappers.proto
   ignore_only:
+    RPC_NO_SERVER_STREAMING:
+      - teleport/auditlog/v1/auditlog.proto
```

Comment:
Did you consider adding that to the ignore item for `UNARY_RPC` below?

OTOH, `RPC_NO_SERVER_STREAMING` is more specific than `UNARY_RPC`. But `UNARY_RPC` already has a comment as to why `UNARY_RPC` is used in the first place. 🤔

Reply:
I missed that. Nice catch.